src/Agent.Worker/JobExtension.cs

```diff
@@ -162,6 +164,30 @@ public async Task<List<IStep>> InitializeJob(IExecutionContext jobContext, Pipel
                         }
                     }
 
+                    //Check if we need to install old node runners
+                    var requiredOldNodeRunnersValue = AgentKnobs.InstallOldNodeRunners.GetValue(context).AsString();
```

Comment:
>AsString [](http://example.com/codeflow?start=105&length=8)

AsBoolean

Reply:
on ADO site it can be set to several runners, ex: "6" or "6:10"